Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -92,7 +92,7 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                     DateHelper.EndMonthUpdater(update.CallbackQuery, ref user);
                     await new SendDayCommand(mediator, telegraBotClient).SendAsync(
                         update.CallbackQuery
-                        ,"Please Select Workplace Booking Type "
+                        , "Please select end date day"
```

Comment:
Why do you need this space after the comma? You could leave the comma on previous line

Reply:
Done